Makefile

```diff
@@ -71,13 +71,21 @@ e2e:
 	go test ./test/e2e/... -coverprofile cover.out -v
 
 .PHONY: e2e-local
-e2e-local: generate-manifests generate envtest ## Run e2e tests.
+e2e-local: generate-manifests generate envtest ## Run e2e tests against mock api.
 	unset USE_EXISTING_CLUSTER && KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./test/e2e/... -coverprofile cover.out -v
 
 .PHONY: e2e-remote
-e2e-remote: 
+e2e-remote: ## Run e2e tests against a remote Greenhouse cluster.
 	USE_EXISTING_CLUSTER=true go test ./test/e2e/... -coverprofile cover.out -v
```

Comment:
Renaming the environment variable is important in this case, as a wrongly scoped `KUBECONFIG` could lead to targeting the wrong cluster.